core/models/asset.py

```diff
@@ -286,3 +286,17 @@ def save_current_asset_owner(sender, **kwargs):
     asset_owner = kwargs.get('instance')
     asset_owner.asset.assigned_to = asset_owner.current_owner
     asset_owner.asset.save()
+
+
+@receiver(post_save, sender=AllocationHistory)
```

Comment:
we have another `post_save` [here](https://github.com/AndelaOSP/art-backend/pull/86/files#diff-bea1207c116ad66b7b398915447bd091R284). We could combine the 2